### PR TITLE
Make clusterrouter deployable

### PR DESCRIPTION
**Pull request by xyz2277**
#### Which issue(s) does this PR fix?

Fixes #
#### Screenshots (if applicable):

```release-note
Make clusterrouter deployable
```

#### Additional Comments:

```release-note

```

### CODE REVIEW

**Comment by xyz2277** [comment]
@ONE7live

File: deploy/clusterrouter.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      app: knode
+  template:
+    metadata:
+      labels:
+        app: knode
+    spec:
+      containers:
+        - name: knode
+          image: nexus.cmss.com:8086/kosmos/clusterrouter/knode:V0.1
+          imagePullPolicy: Always
```
Comment (wangyizhi1) [inline deploy/clusterrouter.yml]: Is `nexus.cmss.com:8086/kosmos/knode:v0.1.0` better?

File: Makefile [hunk 2]
```diff
@@ -104,6 +105,7 @@ upload-images: images
 	docker push ${REGISTRY}/clusterlink-network-manager:${VERSION}
 	docker push ${REGISTRY}/clusterlink-floater:${VERSION}
 	docker push ${REGISTRY}/clusterlink-elector:${VERSION}
+	docker push ${REGISTRY}/clusterrouter:${VERSION}
```
Comment (wangyizhi1) [inline Makefile:109]: clusterrouter -> knode ?

File: deploy/clusterrouter.yml (hunk 1 above)
Comment (wuyingjun-lucky) [reply deploy/clusterrouter.yml]: It is better to modify the `image`

File: deploy/clusterrouter.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
...RUZBQU9DQVE4QU1JSUJDZ0tDQVFFQTBJS0cvQnVXRXJSL0w3QmEKWExmYUZpbVovRTY5VlpTMjNyd2JORWk4U2pSUEs4SE41VU53UXlBNEIwVG1WTkxLYU5vWWVOd2pmWDRWYlVJbwpMY29uU2dudGttamRuc1g1SURDRWY2UjFGT3pzVXFMbFR3aXJ4cjlSNGw0QjdrQi95T1ordFlhQzNUaWFvNGc4Ck9LdkVaU0tWblRvVUptcU5CSHRCZkoxVFU0VE12cmFoaTN2b2YwWWpKR0o4ck1PYmVtWERuOTU1S1JucWlJTGYKZ1ZZY3pOL0pDRldvbEoybkNNeGhMcnNWMlZQakNnN2s3UWZMNDJEUlBZZS9icXN1S01Mc0VoTzVPOTh5Y1JnaApZNkhnV2thazc3U3RmOHNoUnFyRE5vTjhDbnp5S0wrbmdMTEJYUW9tUDkxZHNMOVlEU2FrSkI2SlQ3Ly9xa2RDCnNtVXROUUlEQVFBQm8xWXdWREFPQmdOVkhROEJBZjhFQkFNQ0JhQXdFd1lEVlIwbEJBd3dDZ1lJS3dZQkJRVUgKQXdJd0RBWURWUjBUQVFIL0JBSXdBREFmQmdOVkhTTUVHREFXZ0JRclhiUHhLdDdCSWVEQk9MZG43WkdRcVlVUwpVakFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBZUNROXJ6WGdmd2JaaEUzeXA3bmNxQjlzcFdXdXpjQUFkRDVFCmtXSU1jZ3Rwc1RxTUR5YlVhOUp6WTdRNExtSEI4M3psYTRJWmJNVnFDVTdXTlZFLzZQYU5rNGUzU1Q1NERqbEYKUmFRTjBWMHdQTitIRldOZTkwRlFiSCtYMGp2b3ZEU3JIL2t4WGdLWnFNbTlHMWZRRW9jUG1wWWdpdWUvTi81VwpSdkI4Y1B2RlpMTjVhNG9Wa3NjUEp5Z2dRbzFDbU5PNmFNQkFaYlVmZjR4QjVuL0t2eklDT0pCTTJiK3R3VVNkCmpzcjU2ZkxCYk5MbUNaTWpURDk3SnRrMmgvMmU0dHFUY3dON0RsK2ZpRTdSYm9VMGJmekxBVjFUQXhLbHI2NGQKdFBPbXBDY0xRV0lVWnFIMzNMM3I5VHRBVUtsZHB2SjlvMlo3eEkzNW5XSTVZdWxGOFE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```
Comment (OrangeBao) [inline deploy/clusterrouter.yml]: Submitting certificates seems unsafe

Comment (xyz2277) [reply deploy/clusterrouter.yml]: > Submitting certificates seems unsafe

it's just an arbitrary certificates

**Comment by xyz2277** [comment]
@wangyizhi1 @ONE7live

**Review by wangyizhi1 — APPROVED** [review]

File: deploy/clustertree-knode-manager.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+kind: ConfigMap
+metadata:
+  name: host-kubeconfig
```
Comment (zhouhaoA1) [inline deploy/clustertree-knode-manager.yml]: modiy `host-kubeconfig` should be recorded in installation manual.

**Pull request merged**